paper/config/plugins/SimpleAdminHacks/config.yml

```diff
@@ -454,20 +454,29 @@ hacks:
-       - "   &1FactoryMod&r\nStoke the fires of industry by filling a chest with 8x64 charcoal, placing a crafting table next to the chest, and then a furnace containing more charcoal next to the crafting table. Hit the crafting table with a stick twice, add wood to the chest, and then"
-       - "hit the furnace with a stick. You are now a wealthy charcoal baron capable of producing one of many items faster and more efficiently than those savages still using furnaces! Type \"/fm\" for more info – almost all items have factories."
-       - "   &1RealisticBiomes&r\nYou will starve! Crops and trees require sunlight or glowstone, grow slowly across the board, and do not grow in inappropriate biomes. This encourages trade and warfare because diverse crops are required for FactoryMod XP production."
-       - "   &5Closing Remarks&r\nType /help for more info on everything. Go to https://redd.it/j3nk2f for a more detailed guide. There is much more to learn."
-       - "If you would like to report hacks or glitches, please message the admins through subreddit modmail at reddit.com/r/CivMC\n\nDisclaimer: &2TangentialThreat&r is not staff and should not be trusted."
+       - "&5Hello and welcome to CivMC, &2${player}!\n\n   A server based on nation building and player freedom. Where admin control is minimal, and players dispense their own justice." 
+	   	   - "   &1Namelayer&r\nPocket radios! Create or join chat groups to communicate with friends all over CivMC. By default you have joined global chat or ! chat. You can switch from local chat (1000 block radius) to this one by performing the command: /g !"
+	   - "Go back to localchat by performing the /g command on it own. This works for any group you are allowed to speak in. just switch ! for the name of the group. \nCreate your own group using the GUI by perfoming the /nl Command."
+	  	   - "   &1Citadel&r\nAnother use for your Namelayer groups! Slow down raiders by reinforcing blocks such as your house or your chests and doors to lock them. Reinforced blocks take multiple breaks to drop for players not on the namelayer group."
+	   - "To reinforce a block perform the command /ctr (Groupname) while holding stone, iron or diamond. then punch any block to reinforce it to that group. Now only your allies have ease of access! Tip: Use /ctf instead to reinforce while building!"
+ 	   - "   &1Hiddenore&r\nSudden iron surrounds you! Ores spawn around you as you mine. While caves are still dusted with some visible ores, the main way to get ores is to dig. Ores still hold the same height level for their rarity.
+	   - "Diamonds spawn in veins, where you find one, you'll find others. A message will appear in chat when you've found one. Use silk touch on all ores for increased returns when smelting them in the Ore smelter factory. (better than fortune!)"
```

Comment:
We should add some clarity here as to what a vein means. As a new player, I wouldn't think of finding a single diamond block in isolation a 'vein'. We should describe how these veins are actually quite large in size, can contain hundreds of diamonds, but the ore blocks are separated by as many as 11 blocks so there is more digging required.

> Pro tip! Create waypoints using a client-side mod like JourneyMap when digging up ores in a vein to reveal the overall vein superstructure as you excavate it.